pkg/sql/convergence_test.go

```diff
@@ -29,6 +29,12 @@ func TestConvergenceAndAccuracy(t *testing.T) {
 		t.Skip("only run convergence test with MySQL")
 	}
 	a := assert.New(t)
+
+	// Fix random seed to get deterministic result, avoiding random failure
+	SetTensorFlowRandomSeed(1)
+
+	defer SetTensorFlowRandomSeed(nil)
```

Comment:
Because `SetTensorFlowRandomSeed` is only used here, how about we don't add the new file `env.go`?  Instead, we can move the code here locally.

```go
// Set environment variable which will be read by 
// sqlflow_submitter. get_tf_random_seed to seed TF-related unit tests.
envKey := "SQLFLOW_TF_RANDOM_SEED"
os.Setenv(envKey, 1)
defer os.Unsetenv(envKey)
```

Reply:
Done.